src/screens/ProfileScreen/DetailsScreen/ProgressDetailScreen.tsx

```diff
@@ -10,22 +10,14 @@ import React, { useState } from 'react';
 
 const ProgressDetailScreen = () => {
   // TODO: progress detail api call
-  // const route = useRoute();
 
-  // const { taskId } = route.params;
   const [taskCompleted, setTaskCompleted] = useState('');
   const [taskPlanned, setTaskPlanned] = useState('');
   const [taskBlockers, setTaskBlockers] = useState('');
-  // const [shouldDisable, setShoulDisable] = useState(false);
 
-  const isVisible = () => {
-    // setShoulDisable('');
-    // validateTaskProgress(taskCompleted, taskPlanned, taskBlockers),
-  };
+  const isVisible = () => {};
```

Comment:
Why do we have this?

Reply:
I am assuming if we require this function later to write logic inside it.

Reply:
We can write it whenever it is required. No point including it in this PR when it is not being used.